### PR TITLE
EVG-17909: Add tests for the Logkeeper route handlers

### PR DESCRIPTION
**Pull request by julianedwards**
- Add unit tests for the route handlers
- Some clean up.

### CODE REVIEW

File: views.go [hunk 1]
```diff
@@ -30,11 +30,12 @@ func init() {
 	origins := os.Getenv(corsOriginsEnvVariable)
 	if origins == "" {
 		corsOrigins = []string{}
+		return
 	}
 	corsOrigins = strings.Split(origins, ",")
 }
 
-func addCorsHeaders(w http.ResponseWriter, r *http.Request) {
+func addCORSHeaders(w http.ResponseWriter, r *http.Request) {
```
Comment (johndaniels) [inline views.go:38]: Good styleguide fix.

File: views.go [hunk 2]
```diff
@@ -30,11 +30,12 @@ func init() {
 	origins := os.Getenv(corsOriginsEnvVariable)
 	if origins == "" {
 		corsOrigins = []string{}
+		return
```
Comment (johndaniels) [inline views.go:33]: Good catch.

**Review by johndaniels — CHANGES_REQUESTED** [review]

**Comment by johndaniels** [comment]
Just the one comment. Let me know your thoughts.

File: views.go [hunk 3]
```diff
@@ -37,35 +36,50 @@ func init() {
...
-	opts   Options
+	opts   NewLogkeeperOptions
 }
 
-type createdResponse struct {
-	ID  string `json:"id,omitempty"`
-	URI string `json:"uri"`
+// NewLogkeeperOptions represents the set of options for creating a new
+// Logkeeper REST service.
+type NewLogkeeperOptions struct {
```
[the remark this answers is not in the document]
Comment (julianedwards) [reply views.go:75]: That works for me!

**Review by johndaniels — APPROVED** [review]

**Pull request merged**